lib/game_processor.rb

```diff
@@ -19,15 +19,15 @@ def total_score(value, games)
     return score_sum
   end
 
-  def win_percentage(value, games)
+  def wins_ties(value, games)
     if value == 'home'
       home_wins = 0.0
       games.each do |game|
         if game.home_goals.to_i > game.away_goals.to_i
           home_wins += 1
         end
       end
-      (home_wins / games.count).round(2)
+      home_wins
```

Comment:
nice simplification here and moving the work to a helper method. this simplifies our code